server/server/common/queries.py

```diff
@@ -20,31 +20,30 @@
 _MAX_NESTING_LEVEL = '5'
 
 MENU = '''
-FOR pit IN pitaka
-    FOR div, edge, path IN 1..1 OUTBOUND pit `root_edges` OPTIONS {bfs: false}
-        FILTER IS_SAME_COLLECTION('root', div)
-        LET parents = MERGE(
-            FOR p, p_edge, p_path IN 1..1 INBOUND div `root_edges`
-                RETURN {
-                    [p.type]: {
-                        name: p.name,
-                        uid: p._id,
-                        num: p.num
-                    }
-                }
-            )
-        LET descendents = (
-            FOR d, d_edge, d_path IN 1..100 OUTBOUND div `root_edges`
-                FILTER d_edge.type != 'text'
-                RETURN {
-                    from: d_edge._from,
-                    name: d.name,
-                    uid: d._id,
-                    num: d.num
+FOR div IN root
+    FILTER div.type == 'division'
+    LET parents = MERGE(
+        FOR p, p_edge, p_path IN 1..1 INBOUND div `root_edges`
+            RETURN {
+                [p.type]: {
+                    name: p.name,
+                    uid: p._id,
+                    num: p.num
                 }
-            )
-        
-        RETURN MERGE({uid: div._id, name: div.name, num: div.num}, {descendents: descendents, parents: parents})
+            }
+        )
+    LET descendents = (
+        FOR d, d_edge, d_path IN 1..100 OUTBOUND div `root_edges`
+            FILTER d_edge.type != 'text'
+            RETURN {
+                from: d_edge._from,
+                name: d.name,
+                uid: d._id,
+                num: d.num
+            }
+        )
+    
+    RETURN MERGE({uid: div._id, name: div.name, num: div.num}, {descendents: descendents, parents: parents})
```

Comment:
Why do we need MERE here? Can't we just use 
`{uid: div._id, name: div.name, num: div.num, descendents: descendents, parents: parents}`?